src/libcore/ptr/mod.rs

```diff
@@ -471,10 +474,12 @@ unsafe fn swap_nonoverlapping_bytes(x: *mut u8, y: *mut u8, len: usize) {
 ///
 /// Behavior is undefined if any of the following conditions are violated:
 ///
-/// * `dst` must be [valid] for writes.
+/// * `dst` must be [valid] for reads and writes.
```

Comment:
```suggestion
/// * `dst` must be [valid] for both reads and writes.
```

Reply:
This is missing also in other parts of the docs; changed all.